src/app/pages/arbol-cuentas-contables/movimientos-cuenta/movimientos-cuenta.component.scss

```
@@ -6,7 +6,25 @@
     }
 }
 
-.modal-title {
+:host /deep/ ng2-st-tbody-edit-delete {
```

(En general) Usar `::ng-deep` o [mejor, evitarlo](https://dev.to/jwp/angular-10-ng-deep-and-host-pseudo-classes-1kbg)
```suggestion
:host ::ng-deep ng2-st-tbody-edit-delete {
```